core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -73,8 +73,10 @@
 H/15 * * * *
 # every ten minutes in the first half of every hour (three times, perhaps at :04, :14, :24)
 H(0-29)/10 * * * *
-# once every two hours every weekday (perhaps at 9:38 AM, 11:38 AM, 1:38 PM, 3:38 PM)
-H 9-16/2 * * 1-5
+# once every two hours at 45 minutes past the hour starting at 9:45 AM and finishing at 3:45PM every weekday.
+H 9-16/2 45 * 1-5
```

Comment:
:bug: This will run on the 45th day of the month, i.e., never. Did you mean

    45 9-16/2 * * 1-5

?